suites/message/nested.go

```diff
@@ -59,11 +59,9 @@ func MessageTest_NestedSends(t *testing.T, factory state.Factories) {
 		stage := prepareStage(td, acctDefaultBalance, multisigBalance)
 		balanceBefore := td.GetBalance(stage.creator)
 
-		// Multisig sends back to the creator.
-		amtSent := abi.NewTokenAmount(1)
-		result := stage.sendOk(stage.creator, amtSent, builtin.MethodSend, nil, nonce)
+		result := stage.sendOk(builtin.BurntFundsActorAddr, abi.NewTokenAmount(1), builtin.MethodSend, nil, nonce)
 
-		td.AssertActor(stage.creator, big.Sub(big.Add(balanceBefore, amtSent), result.Receipt.GasUsed.Big()), nonce+1)
+		td.AssertActor(stage.creator, balanceBefore, result.Receipt, nonce+1)
```

Comment:
This is now missing the essential assertion that balance was transferred from the multisig to the destination!

Reply:
Fixed, thanks!